src/east/workspace_commands/codechecker_commands.py

```diff
@@ -0,0 +1,202 @@
+    """
+    east.pre_workspace_command_check(check_only_west_workspace=True)
+
+    cc = east.consts["codechecker_path"]
+    cfg = os.path.join(east.project_dir, "codechecker_config.yaml")
+    compile_commands = os.path.join("build", "compile_commands.json")
+
+    check_for_compile_commands_json(east, compile_commands)
+    check_for_codechecker_config_yaml(east, cfg)
+
+    if not skip_file:
+        skip_file = create_skip_file(east, CC_OUTPUT_DIR)
+
+    # Run analyze command
+    analyze_cmd = (
+        f"{cc} analyze --skip {skip_file} --output {CC_OUTPUT_DIR} "
+        f"{compile_commands} --config {cfg}"
+    )
+
+    east.run(analyze_cmd)
+
+    if not dont_cleanup_plist:
+        cleanup_plist_files(east, CC_OUTPUT_DIR)
+
+    if only_analyze:
+        return
+
+    # Run parse command
+    parse_cmd = f"{cc} parse --config {cfg} {CC_OUTPUT_DIR } {html} "
+
+    if html:
+        parse_cmd += f"--output {CC_OUTPUT_DIR } --export html "
+
+    result = east.run(parse_cmd, exit_on_error=False, return_output=True, silent=True)
+
+    # Print output of the parse command, with default rich highlighting, but without any
+    # other wrapping, cropping.
+    print_args = {
+        "highlight": True,
+        "overflow": "ignore",
+        "crop": False,
+        "soft_wrap": False,
+        "no_wrap": True,
+    }
+    east.print(result["output"], **print_args)
+
+
+@click.option(
+    "-a",
+    "--apply",
+    is_flag=True,
+    help="Apply the available automatic fixes. "
+    "This causes the modification of the source code. Default: false",
+)
+@click.command(**east_command_settings)
+@click.pass_obj
+def fixit(east, apply):
+    """Apply fixes suggested by the [magenta bold]Codechecker[/].
+
+    \b
+    \n\nSome analyzers may suggest some automatic bugfixes. Most of the times these are style issues which can be fixed easily. This command handles the listing and application of these automatic fixes.
+
+    \n\n[bold]Note:[/] This command should be ran after the [bold cyan]east codechecker check[/] command.
+
+    \n\n[bold]Note:[/] This command can be only run from inside of a [bold yellow]West workspace[/].
+    """
+
+    east.pre_workspace_command_check(check_only_west_workspace=True)
+
+    cc = east.consts["codechecker_path"]
+
+    fixit_cmd = f"{cc} fixit {CC_OUTPUT_DIR}"
+
+    if apply:
+        fixit_cmd += " --apply"
+
+    east.run(fixit_cmd)
+
+
+example_config_text = """
+analyzer:
+  - --keep-gcc-include-fixed
+  - --keep-gcc-intrin
+  # Uncomment below option when you want to use .clang-tidy file.
+  # - --analyzer-config=clang-tidy:take-config-from-directory=true
+  - --analyzers
+  - clang-tidy
+  - clangsa
+  - --enable=guideline:sei-cert
+  # Enable for cross translation unit analyis, but analysis will take longer.
+  # - --ctu
+
+parse:
+  - --trim-path-prefix=/*/project
+  - --print-steps
+"""
+
+
+@click.command(**east_command_settings)
+@click.pass_obj
+def example_config(east):
```

Comment:
You could add a `--create` flag here that would create the file and paste the example config into it.

Reply:
Good idea, will do that